code/game/objects/items/weapons/material/misc.dm

```diff
@@ -68,7 +68,7 @@
 	set_extension(src, /datum/extension/tool, list(TOOL_HATCHET = TOOL_QUALITY_DEFAULT))
 
 /obj/item/hatchet/unbreakable
-	unbreakable = TRUE
```

Comment:
That's not really accurate though? All the unbreakable things are used in borg inventories. They shouldn't ever need to take damages since they're not supposed to exist outside the borg.

Reply:
Fair point. I'm guessing its to prevent shattering on use? I could probably set some flag for preventing damage from wear instead of making them completely invulnerable?

Reply:
That would mean people could reasonably assumed that unbreakable is synonym with not taking damage. And then use it by mistake instead of the actual way of making things not take damage?

Reply:
Had to do a bit more than that since the var turned out to be handling both wear and shattering on destruction/usage.
But I restored the functionality without making those things invulnerable to damage.